addons/mail/static/src/models/messaging/messaging.js

```diff
@@ -297,6 +297,10 @@ function factory(dependencies) {
             readonly: true,
             required: true,
         }),
+        /**
+         * Id of the current user's res.users.settings record.
+         */
+        resUsersSettingsId: attr(),
```

Comment:
Would either move this field in user or rename so that we get it's from the current user just at a glance on the field name (e.g. "currentUserResUsersSettingsId").

Long name is essentially caused by lack of context (current user) and too-technical-for-my-taste concept (res users settings id). The former is appropriately solve by moving field in user model, hence the preferred approach is moving to User model.

Reply:
moving to user then